mypyc/test-data/run-strings.test

```diff
@@ -617,3 +617,9 @@ def test_encode() -> None:
         assert False
     except UnicodeEncodeError:
         pass
+
+[case testStringMin]
+def test_str_min() -> None:
+    x: str = 'aaa'
+    y: str = 'bbb'
+    assert min(x, y) == 'aaa'
```

Comment:
you can also test the minimun of user defined class which has `__lt__`.

Reply:
Added

Reply:
Test the inverse here as well.